src/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import BigNumber from 'bignumber.js';
+
+@Injectable()
+export class Utils {
+  constructor() {
+    BigNumber.config({
+      EXPONENTIAL_AT: [-30, 30],
+    });
+  }
+
+  async wait(ms) {
+    return new Promise((resolve) => {
+      setTimeout(resolve, ms);
+    });
+  }
+
+  getAmountValue(strNum: string) {
```

Comment:
That's method don't depend on internal class state and will be better use static method instead class method.

Reply:
Fixed